src/diamond/handler/netuitive_handler.py

```diff
@@ -277,32 +277,36 @@ def _add_aws_meta(self):
             resp = urllib2.urlopen(request, timeout=1).read()
             j = json.loads(resp)
 
-            for k, v in j.items():
-                if type(v) is list:
-                    vl = ', '.join(v)
-                    v = vl
-                self.element.add_attribute(k, v)
+            if j and self.aws_meta_state < 1:
```

Comment:
Why not use a boolean?

Reply:
Is there a benefit to using a boolean over the current implementation?

Reply:
The variable's purpose is easier to understand. I searched for a few seconds to see other references for this variable to see if other values had other interpretations.